src/components/Notification.vue

```diff
@@ -39,8 +39,8 @@ export default {
   max-width: rem(520px);
   margin-left: auto;
   margin-right: auto;
-  padding-top: env(safe-area-inset-top);
-  border-radius: 0 0 rem(8px) rem(8px);
+  margin-top: calc(#{rem(10px)} + env(safe-area-inset-top, 0));
```

Comment:
with `env` this property can be ignored by some browsers, check how we already doing it